src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -224,8 +224,17 @@ bool WglContext::makeCurrent(bool current)
 ////////////////////////////////////////////////////////////
 void WglContext::display()
 {
-    if (m_deviceContext && m_context)
-        SwapBuffers(m_deviceContext);
+    if (m_deviceContext && m_context) 
+    {
+        bool success = SwapBuffers(m_deviceContext);
+        if (!success) 
+        {
+            err() << "SwapBuffers error: " << GetLastError() << std::endl;
+            ReleaseDC(m_window, m_deviceContext);
+            m_deviceContext = GetDC(m_window);
+            success = SwapBuffers(m_deviceContext);
```

Comment:
This is a dead store. `success` is never read after this. Let's not store this boolean at all since we don't need it. Also formatting is slightly incorrect. I tried to fix this and force-update the branch but I seem to not be allowed to do that so please go check out the Formatting CI job and make the changes it requests.